### ISSUE TITLE
twin.macro: Cannot read property 'name' of null

### ISSUE THREAD

**Issue by sotrustmtg** [ben-rogerson/twin.macro#113]
I am getting this error on 'gatsby develop'
**Generating development JavaScript bundle failed**
**/project/src/pages/index.tsx: twin.macro: Cannot read property 'name' of null**
**File: src/pages/index.tsx**
**failed Re-building development bundle - 0.184s**

twin.macro v.1.6.0

```
import * as React from 'react'
/** @jsx jsx */
import { css, jsx } from '@emotion/core'
import styled from '@emotion/styled'
import tw, { theme } from 'twin.macro'
// removed for support ...

const IndexPage = () => {
  return (
    <IndexLayout>
      <div
        tw="items-center justify-center self-center flex flex-col  lg:px-4 lg:mx-auto lg:flex-row"
        css={css`
          ${mq.xl} {
            width: 1024px;
          }
        `}
      >
        <div css={tw`w-full lg:w-1/2 flex flex-col justify-center`}>
          <div css={tw`p-4`}>
            <H1>Header</H1>
            <H3>Header!</H3>
            <P>some text</P>
            <P>some more text</P>
            <div
              css={[
                tw`flex flex-row justify-start`,
                css`
                  width: 430px;
                `
              ]}
            >
              <LinkButton
                to="/home-loan/"
                onKeyUp={e => {
                  return (e.keyCode === 13 || e.keyCode === 32) && (() => navigate('/next/'))
                }}
                tabIndex="0"
                role="button"
              >
                Start Quiz
              </LinkButton>
            </div>
          </div>
        </div>
        <div css={tw`lg:w-1/2 pt-2`}>
          <img src={img} alt=" " />
        </div>
      </div>
    </IndexLayout>
  )
}

export default IndexPage

const H1 = styled.h1`
  font-family: 'rift', sans-serif;
  color: #1d1d1d;
  ${tw`text-3xl lg:${theme.fontSize.lg41} uppercase`}
  margin-bottom:10px;
`
const H3 = styled.h3`
  font-family: 'rift', sans-serif;
  color: #00b77b;
  ${tw`text-3xl uppercase`}
  margin:0 0 10px;
  font-size: 42px;
`

const P = styled.p`
  color: #1d1d1d;
  font-family: 'proxima-nova', sans-serif;
  font-weight: 500;
  font-size: 24px;
`
```

**Comment by sotrustmtg** [comment]
removing the theme-related items compiles just fine.

**Comment by sotrustmtg** [comment]
removing the following from pakage.json resolved the issue also:
```
"babelMacros": {
    "twin": {
      "config": "tailwind.config.js",
      "preset": "emotion",
      "debugProp": true,
      "debugPlugins": false,
      "debug": false
    }
  },
```

**Comment by ben-rogerson** [comment]
Looks like the debugProp is causing some issues at the moment, it triggers an error when there's a `css` and `tw` on a single jsx element.
I'll patch that shortly.

In the meantime, turn off debugProp in your twin config:

```js
"debugProp": false,
```

**Comment by ben-rogerson** [comment]
[Fixed in v1.6.1 🎉 ](https://github.com/ben-rogerson/twin.macro/releases/tag/1.6.1)

Thanks for reporting the issue 👍
